s3store/s3store.go

```diff
@@ -148,7 +149,7 @@ func New(bucket string, service S3API) S3Store {
 	return S3Store{
 		Bucket:      bucket,
 		Service:     service,
-		MaxPartSize: 6 * 1024 * 1024,
+		MaxPartSize: 5 * 1024 * 1024 * 1024,
```

Comment:
@koenvo good point!

While I would propose to solve that differently, as going your way causes confusion, since the S3 API spec states a MaxPartSize of "5 * 1024 * 1024 * 1024" and new contributors will not intuitively understand, why that is different here. 

I would therefore propose to add two other variables here and adapt calcPartSize take them into account, which - as you correctly found out - is not the case now as far as MaxUploadSize goes.

```
MaxUploadParts: 10000
 MaxUploadSize: 5 * 1024 * 1024 * 1024 * 1024
```